.github/workflows/lock-threads.yml

```diff
@@ -57,15 +58,20 @@ jobs:
             After 60 days of no activity, we'll close this PR. Keep in mind, I'm just a robot, so if I've closed this PR in error, please reply here and my human colleagues will reopen it.
 
             Thanks for being a part of the Clerk community! 🙏
-      - uses: actions/stale@v8
+      - uses: actions/stale@v9
+        name: Stale action for needs-reproduction issues
         with:
           repo-token: ${{ secrets.CLERK_COOKIE_PAT }}
           days-before-issue-stale: 7
           days-before-issue-close: 1
           exempt-all-assignees: true
-          only-issue-labels: 'needs-reproduction'
+          any-of-issue-labels: 'needs-reproduction'
```

Comment:
https://github.com/actions/stale#only-issue-labels

> process the issues that contain all these label(s)

So only issues with _only_ `needs-reproduction` would be touched, not issues with `sdk` + `needs-reproduction`.

https://github.com/actions/stale#any-of-issue-labels

> process the issues that contain one of these label(s)